agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1582,6 +1585,18 @@ bool main_thread::handle_1905_discovery_query(ieee1905_1::CmduMessageRx &cmdu_rx
     return true;
 }
 
+/**
+ * @brief Handles 1905 Discovery Query message, currently only prints to the log
+ * @param cmdu_rx
+ * @return true on success
+ * @return false on failure
+ */
+bool main_thread::handle_1905_client_capability_query(ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    LOG(DEBUG) << "Received Client capability Query message";
```

Comment:
Ok we received capability query. 
Why didn't create a response? If you don't know what to fill the TLVs in the response, use dummy values.

Reply:
No need for a response, it is enough to just verify the request was received.